lib/league_statistics.rb

```diff
@@ -0,0 +1,100 @@
+require_relative "./stat_tracker"
+require "csv"
+
+class LeagueStatistics < StatTracker
+
+  def count_of_teams
+    total_games = CSV.read(@teams, :headers=>true)
+    total_games.count
+  end
+
+  def team_name(id)
```

Comment:
@ajtran303  @Arique1104  @Jonathan-M-Wilson This is my way of converting the team ID to the team name. I think this could be a beneficial StatTracker Class method. Let me know if anyone else has to convert team_id to a name